lib/mix/lib/mix/tasks/escript.build.ex

```diff
@@ -123,6 +126,7 @@ defmodule Mix.Tasks.Escript.Build do
     filename     = escript_opts[:path] || script_name
     main         = escript_opts[:main_module]
     app          = Keyword.get(escript_opts, :app, project[:app])
+    strip_beam   = Keyword.get(escript_opts, :strip_beam, true)
```

Comment:
I think `strip` is more correct in this case since we are not adjusting it but rather removing part of it.